vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -98,6 +100,23 @@
 namespace react {
 namespace uwp {
 
+namespace {
+
+bool HasPackageIdentity() noexcept {
+  static const bool hasPackageIdentity = []() noexcept {
+    try {
+      auto package = winrt::Windows::ApplicationModel::Package::Current();
+      return true;
+    } catch (...) {
```

Comment:
>... [](start = 13, length = 3)

catch(...) is generally a bad practice.  You should be able to catch a specific type of exception here.

Reply:
I could catch a `winrt::hresult_error const&`, and check that the error is the one I expect, sure.

Reply:
Coates reminded me that I should know better than to cause exceptions to be thrown... so I'm doing the crazy incantations to get down to the ABI. I'm not sure why that's so much harder than `winrt::get_abi(winrt::get_activation_factory<Foo, IFooStatics>())->get_Current(...)` but I'm sure there are good reasons...